lib/entity.js

```diff
@@ -10,16 +9,39 @@ function Entity (entity) {
   this.type = entity.type || 'Unknown';
   this.name = entity.name || this.id;
 
-  this.experience = 0;
+  this.EXP = 0;
+  this.statPoints = 15;
 
-  this.equipment = {
-    armor: null,
-    weapon: null
+  this.stats = {
+    attack: 0,
+    agility: 0,
+    vitality: 0,
+    invocation: 0,
+    influence: 0,
+    shielding: 0,
+    smarts: 0,
+    willpower: 0,
+    charm: 0
   };
 
-  this.stats = {
-    strength: 0,
-    luck: 0
+  this.derrivedStats = {
+    damage: 0,
+    hp: 0,
+    mDamage: 0,
+    enchantments: 0,
+    mHP: 0,
+    library: 0,
+    ap: 0,
+    summons: 0
+  };
+
+  this.cardLibrary = {
+    attackCards: 0,
+    itemCards: 0,
+    summonCards: 0,
+    spellCards: 0,
+    enchantCards: 0,
+    leylineCards: 0
```

Comment:
These two new properties seem like they should be simplified and variable names thought about a bit more.

Reply:
Those are meant to be "maximum" variables, so the names should likely be changed to represent this, or they should be condensed -- or maximums removed entirely. Not sure.